en/user/articles-dxp/120-search/13-result-rankings.markdown

```diff
@@ -74,6 +74,22 @@ ranking under a different name. See
 [LPS-96357](https://issues.liferay.com/browse/LPS-96357)
 for more information.
 
```

Comment:
@mtokudome, this info (feel free to reorganize and adjust) can go into the Results Rankings article. :point_down:

Reply:
@lipusz Makoto has added these details already, in his port of the synonyms and result rankings articles.. We'll incorporate the other pieces into the upgrading guide.